cves/kernel/CVE-2020-11884.yml

```diff
@@ -55,15 +55,17 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  In the Linux kernel 4.19 through 5.6.7 on the s390 platform, code was executing 
+  because of a race condition. Failure to protect a concurrent page table upgrade.
```

Comment:
I'm confused about what is actually vulnerable about this vulnerability. Is it an Integrity problem?  What was the code that was executing doing?

Reply:
The race condition would lead to the crashing of the application. This has been added.